docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.hero.scss

```diff
@@ -40,33 +38,47 @@
     margin: calc(-1 * #{hb-spacing-width()}) 0 hb-spacing-width();
   }
 
-  @include hb-colorful {
-    position: relative;
-    min-height: 60vh;
+  position: relative;
+  min-height: 60vh;
 
+  @include hb-themes(('colorful', 'airy')) {
     @include grid-media-min('sm') {
       min-height: 55vh;
     }
+  }
 
-    &::before {
-      @include hb-pairing-color('background-color', 'primary-hero-overlay');
-      content: "";
-      position: absolute;
-      top: 0;
-      left: 0;
-      width: 100%;
-      height: 100%;
-      z-index: $hb-z-index-hero-fallback;
+  .hb-three-column--one-sidebar & {
```

Comment:
cool!

Reply:
what about for two-sidebars?

Reply:
I created a page to see what happens to the hero and carousel when there are two sidebars. The hero still looks great. :tada: But the carousel needed some love. The min-height of the image was fine but the content was swished. So I adjusted the margins and widths of content when there are two sidebars. :smile:

Reply:
looks great!!!